tests/test_seqsandresiduepairs.py

```
@@ -0,0 +1,395 @@
+
+import json
+import logging
+import os
+import shutil
+import tempfile
+from parser.process_dataset import (
+    json_sequences_and_residue_pairs,
+    sequences_and_residue_pairs,
+)
+
+import orjson
```

Flake8 (7.3.0)

[error] 'orjson' imported but unused

(F401)